production/helm/loki/templates/backend/statefulset-backend.yaml

```diff
@@ -23,7 +23,7 @@ spec:
   {{- if eq .Values.deploymentMode "SingleBinary" }}
   replicas: 0
   {{- else }}
-  replicas: {{ .Values.write.replicas }}
+  replicas: {{ .Values.backend.replicas }}
```

Comment:
4 lines above this still says `if not .Values.write.autoscaling.enabled`. Shouldn't that use backend as well?

Reply:
Good point